src/main/java/ch/wisv/areafiftylan/extras/mailupdates/model/Subscription.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2017  W.I.S.V. 'Christiaan Huygens'
+ *
+ *      This program is free software: you can redistribute it and/or modify
+ *      it under the terms of the GNU General Public License as published by
+ *      the Free Software Foundation, either version 3 of the License, or
+ *      (at your option) any later version.
+ *
+ *      This program is distributed in the hope that it will be useful,
+ *      but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *      GNU General Public License for more details.
+ *
+ *      You should have received a copy of the GNU General Public License
+ *      along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ch.wisv.areafiftylan.extras.mailupdates.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.validator.constraints.Email;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+/**
+ * Used to gather emails from the frontend, to allow interested users
+ * to receive emails about the event.
+ *
+ * @author Jurriaan Den Toonder Created on 23-10-17
+ */
+@Entity
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor
+public class Subscription {
```

Comment:
I just thought about this, it is very easy to allow users to unsubscribe, just add an active flag to the subscription.

Reply:
Discussed, keeping it like this